code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -124,7 +124,7 @@ There are several things that need to be remembered:
 		else if(U.adjusted == DIGITIGRADE_STYLE)
 			t_color = "[t_color]_l"
 
-		var/mutable_appearance/uniform_overlay
+		var/mutable_appearance/uniform_overlay = overlays_standing[UNIFORM_LAYER]

```

Comment:
You changed the default here so the uniform won't update for species without gender variants.